ansible/ansible_collections/agentic/agent_configs/plugins/modules/deepseek_tui_hook.py

```
@@ -0,0 +1,383 @@
+            hooks_root.pop("hooks", None)
+        if not hooks_root:
+            data.pop("hooks", None)
+
+    if changed or removed_legacy_block:
+        if module.check_mode:
+            return True, data, existed_before
+        write_toml_if_changed(module, path, data)
+    return changed, data, existed_before
+
+
+def state_transition(changed: bool, existed_before: bool, state: str) -> str:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)